Userland/Libraries/LibWeb/Loader/ResourceLoader.cpp

```diff
@@ -4,6 +4,15 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
+// FIXME: LibIPC Decoder and Encoder are sensitive to include order here
+// clang-format off
+#include <WebContent/ConnectionFromClient.h>
```

Comment:
LibWeb should not know anything about `WebContent`. Please use the `Web::PageClient` interface to communicate between LibWeb and with the WebContent code.